README.md

```diff
@@ -2,6 +2,12 @@ This is intended to be used by an organization for all of their own accounts. Th
 
 This restriction is put in place by creating a unique role for each account, then attaching an assumerole policy that trusts the corresponding account to assume it. You will still need to give permission to assume roles to your users/roles that are used to apply terraform in other accounts, and configure your state appropriately to use this.
 
+As of v3.0, all public access is blocked by default.  There are individual parameters which can be set to "false" if public bucket/state access is desired:
```

Comment:
@chrischildresssg Thoughts on a single bool here? Is there a time where we need to toggle each specific one. Another option is a single var toggle with a conditional to determine the logic in it.

Reply:
Well, everything is "true" by default. The idea here is that if you specifically needed to allow public ACLs but not the rest, you'd only need to toggle one parameter.  Do you think it makes more sense to have it be "all or nothing"?

Reply:
I like having the option to toggle some and not all at the same time, like how it works in the aws console